apps/console/src/components/pages/auth/subscribe/subscribe.tsx

```diff
@@ -113,7 +113,7 @@ export const Subscribe = () => {
                   {isPending && <LoaderCircle className="animate-spin mr-2" size={20} />}
```

Comment:
Might need to be smaller to not kick everything down a line? Not positive though
```suggestion
                  {isPending && <LoaderCircle className="animate-spin mr-2" size={16} />}
```